lib/src/chart/scatter_chart/scatter_chart_painter.dart

```diff
@@ -290,6 +290,67 @@ class ScatterChartPainter extends AxisChartPainter<ScatterChartData> {
       );
     }
 
+    if (data.scatterLabelSettings.showLabel) {
+      for (final scatterSpot in data.scatterSpots) {
+        int spotIndex = data.scatterSpots.indexOf(scatterSpot);
+
+        String label = data.scatterLabelSettings.getLabelFunction(spotIndex);
+
+        if (label.isEmpty || !scatterSpot.show) {
+          continue;
+        }
+
+        final span = TextSpan(
+          text: label,
+          style: data.scatterLabelSettings.getLabelTextStyleFunction(spotIndex),
+        );
+
+        final tp = TextPainter(
+          text: span,
+          textAlign: TextAlign.center,
+          textDirection: TextDirection.ltr,
```

Comment:
It's better not to hardcode the `textDirection` property because some users have RTL-based language.
You can put a single property in `ScatterLabelSettings` to hold a dynamic value of `textDirection` (with `ltr` as default)